test/acceptance/features/serviceBinding.feature

```diff
@@ -165,3 +165,77 @@ Feature: Bind application to the secret pushed by agent app controller
                 kind: Deployment
         """
         And On Primaza Cluster "main", file "/bindings/newapp-binding/username" is unavailable in application pod running in namespace "applications"
+
+    Scenario: Service binding projection works for multiple applications matching the labels
```

Comment:
seems to me the title is not up to date. Isn't it?

Reply:
`Application Agent watches ServiceBindings' resources and projection works for application created after ServiceBinding creation` ?

Reply:
looks good